Sources/MlemMiddleware/API Client/ApiClient+Post.swift

```diff
@@ -68,15 +68,94 @@ extension ApiClient: PostFeedProvider {
         return nil
     }
     
+    /// Mark the given post as read. Works on all versions.
+    /// On v0.19.0 and above, calling this will also mark any queued posts as read unless `includeQueuedPosts` is set to `false`.
+    public func markPostAsRead(
+        id: Int,
+        read: Bool = true,
+        includeQueuedPosts: Bool = true,
+        semaphore: UInt? = nil
+    ) async throws {
+        // We *must* use `postId` in 0.18 versions, and we *must* use `postIds` from 0.19.4 onwards.
+        let version = try await version
+        let request: MarkPostAsReadRequest
+        if version >= .v19_0 {
```

Comment:
Version inconsistent with comment on L79

Reply:
Between 0.19.0 and 0.19.4, _either_ `postId` or `postIds` can be used. I've opted to use `postIds` as soon as it's possible to do so. I'll update the comment to make this more clear 👍